api/src/database/games/Games.ts

```diff
@@ -19,13 +19,16 @@ export const gameForSlug = (slug: string) => {
     });
 };
 
-export const createGame = (
+export const createGame = async (
     name: string,
     slug: string,
     coverImage?: string,
     owners?: string[],
     moderators?: string[],
 ) => {
+    if (await gameForSlug(slug)) {
```

Comment:
Would it be better to try...catch the prisma error here rather than doing the check? While the rest of the database layer largely doesn't have protections, there are errors outside of our control that can get thrown that we may want to blanket catch and send a 500 response to. If we end up doing that it's more performant to rely on the thrown error rather than hit the database twice, especially as things scale up. Granted, this is a relatively uncommon workflow, so maybe we can overlook the performance concern